solidity/dashboard/src/pages/ApplicationsPageContainer.jsx

```diff
@@ -1,25 +1,30 @@
 import React from "react"
 import { Route, Switch, Redirect } from "react-router-dom"
-import ApplicationOverviewPage from "./ApplicationOverviewPage"
 import TBTCApplicationPage from "./TBTCApplicationPage"
 import KeepRandomBeaconApplicationPage from "./KeepRandomBeaconApplicationPage"
+import PageWrapper from "../components/PageWrapper"
 
+const subLinks = [
+  { title: "Random Beacon", path: "/applications/random-beacon" },
+  { title: "tBTC", path: "/applications/tbtc" },
+]
 const ApplicationsPageContainer = (props) => {
   return (
-    <Switch>
-      <Route
-        exact
-        path="/applications/overview"
-        component={ApplicationOverviewPage}
-      />
-      <Route exact path="/applications/tbtc" component={TBTCApplicationPage} />
-      <Route
-        exact
-        path="/applications/random-beacon"
-        component={KeepRandomBeaconApplicationPage}
-      />
-      <Redirect to="/applications/overview" />
-    </Switch>
+    <PageWrapper title="Applications" subLinks={subLinks}>
+      <Switch>
+        <Route
+          exact
+          path="/applications/tbtc"
+          component={TBTCApplicationPage}
+        />
+        <Route
+          exact
+          path="/applications/random-beacon"
+          component={KeepRandomBeaconApplicationPage}
+        />
```

Comment:
Might be useful to specify these in the same order as the nav lists them.

I also immediately wonder if we could use the `subLinks` data to build this `Route` table, instead of having to specify the info in it twice 🤔 

Perhaps `PageWrapper` could actually take a `{title, path, component}` array and handle the `Route` piece directly? We could always use the first item as the default for the fallthrough below. Then we could probably eliminate a lot of these container files altogether in favor of a single declarative JS object map that describes the page hierarchy and component mapping..? Sounds a little crazy, but it also feels doable…

Reply:
yeah sounds good.

> Sounds a little crazy, but it also feels doable…

 I'll do my best 😄